docs/learn-governance.md

```diff
@@ -261,20 +261,22 @@ against - equate to a simple majority-carries system at 100% turnout.
 
 ## Council
 
-To represent passive stakeholders, we introduce the idea of a "council". The council is an on-chain
-entity comprising a number of actors, each represented as an on-chain account. For Polkadot this
-number is likely to begin at around six people, and increase over the course of 9 months to 24
-people (roughly one extra individual coming on every two weeks). In general, it has a fixed number
-of seats (24 on Polkadot and 17 on Kusama).
+To represent passive stakeholders, Polkadot introduces the idea of a "council". The council is an
+on-chain entity comprising a number of actors, each represented as an on-chain account. On Polkadot
+the council currently consists of 13 members, however this is expected to increase over the next few
+months to 24 people. In general, the council will end up having a fixed number of seats. On
```

Comment:
```suggestion
months to 24 seats. In general, the council will end up having a fixed number of seats. On
```

Reply:
The "in general" sentence is superfluous I think. Also recommend changing people to seats, since there's already people who hold more than one seat.